packages/experimental/Switch/src/Switch.types.ts

```diff
@@ -155,18 +156,21 @@ export interface SwitchProps extends Omit<PressablePropsExtended, 'onPress'> {
 
   /**
    * The Switch's text that shows when it is in a false state
+   * Note: Not supported for Android
    */
   offText?: string;
 
   /**
    * The Switch's text that shows when it is in a true state
+   * Note: Not supported for Android
    */
   onText?: string;
 
   /**
    * The position of the label relative to the Switch. The position value 'after' is mutually
    * exclusive with the onText and offText props. This is due to variable width
    * of the text props causing the Switch's position to change when it shouldn't.
+   * Note : 'before' , 'above' are not supported on Android
```

Comment:
minor nit's on removing the extra spaces.

Reply:
Removed.